### ISSUE TITLE
The toggle switch is not properly associated with its helper text.

### ISSUE THREAD

**Issue by lf-collins** [vmware/clarity#4475]
## How to reproduce

1.  Go to 'https://clarity.design/documentation/toggle-switches'.
2.  Scroll down to "Full toggle switch example" under Helper and error messages section.
3.  Inspect the toggle switch.
4.  See error, the aria-describeby of the toggle switch element does not point to the helper text.
<img width="1165" alt="Screen Shot 2020-04-16 at 4 39 19 PM" src="https://user-images.githubusercontent.com/26111301/79435479-49858b00-8002-11ea-866f-149b9eb17a11.png">

## Expected behavior

The helper text id should not be "null-helper", and the toggle element and its helper should be associated correctly.

## Versions

**App**

* Clarity: [e.g. 3.0]

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

## Additional notes

_Add any other notes about the problem here._

**Comment by KingMario** [comment]
If no `clr-control-helper` is presented, the aria-describedby should be omitted. Is there any way to check if the content is projected?

**Comment by gnomeontherun** [comment]
This is not an issue with the Clarity Core toggle fields, so that is the recommendation to update to using Clarity Core form components in this case. @bdryanovski is looking at fixing this as well for Clarity Angular.
